<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1929</width>
    <height>1102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hexagonal Map</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1789</x>
      <y>0</y>
      <width>141</width>
      <height>800</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="step_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="buy_t1_unit_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Пехота:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="endTurn_pushButtin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>720</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Franklin Gothic Medium</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Конец хода</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Лучники:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Кавалерия:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Нанять (5 золота)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Нанять (10 золота)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Нанять (15 золота)</string>
     </property>
    </widget>
    <widget class="QLabel" name="gold_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ваше золото:</string>
     </property>
    </widget>
    <widget class="QLabel" name="myGoldLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>169</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Доступные рекруты</string>
     </property>
    </widget>
    <widget class="QLabel" name="UnitT1Label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>320</y>
       <width>49</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="UnitT2Label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>390</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="UnitT3Label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>450</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1929</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
